### PR TITLE
Fix 'get_filename' call in 'ninjo' writer

### PR DESCRIPTION
**Pull request by loreclem**
- [ ] Closes #xxxx 
 - [ ] Tests added 
 - [ ] Tests passed 
 - [ ] Passes ``git diff origin/master -- "*py" | flake8 --diff`` 
 - [ ] Fully documented

### CODE REVIEW

File: satpy/writers/ninjotiff.py [hunk 1]
```diff
@@ -49,5 +49,5 @@ def save_image(self, img, filename=None, **kwargs):  # floating_point=False,
 
         .. _ninjotiff: http://www.ssec.wisc.edu/~davidh/polar2grid/misc/NinJo_Satellite_Import_Formats.html
         """
-        filename = filename or self.get_filename(**img.info)
+        filename = filename or self.get_filename()
```
Comment (djhoese) [inline satpy/writers/ninjotiff.py]: Shouldn't this be `**img.attrs`?

Comment (djhoese) [reply satpy/writers/ninjotiff.py]: Sorry:

```
filename = filename or self.get_filename(**img.data.attrs)
```

Comment (loreclem) [reply satpy/writers/ninjotiff.py]: Yes, you are right, I have just tested it. I update this PR right away, thanks for checking.

**Comment by mraspaud** [comment]
Can we merge this @loreclem ?

**Comment by loreclem** [comment]
I would say so, David approved it too.

**Pull request merged**